src/auth/googleauth.ts

```diff
@@ -96,9 +97,7 @@ export interface GoogleAuthOptions {
 export const CLOUD_SDK_CLIENT_ID =
   '764086051850-6qr4p6gpi6hn506pt8ejuq83di341hur.apps.googleusercontent.com';
 
-export class GoogleAuth {
-  transporter?: Transporter;
-
+export class GoogleAuth extends AuthClient {
```

Comment:
this adds setCredentials, inherited by AuthClient, which wouldn't be used.

I think its implementation would need to be
```
this.getClient().setCredentials(...)
```